src/test/java/in/projecteka/consentmanager/dataflow/DataFlowRequesterTest.java

```diff
@@ -2,11 +2,7 @@
 
 import in.projecteka.consentmanager.clients.ClientError;
 import in.projecteka.consentmanager.clients.ConsentManagerClient;
-import in.projecteka.consentmanager.dataflow.model.AccessPeriod;
-import in.projecteka.consentmanager.dataflow.model.ConsentArtefactRepresentation;
-import in.projecteka.consentmanager.dataflow.model.ConsentStatus;
-import in.projecteka.consentmanager.dataflow.model.DateRange;
-import in.projecteka.consentmanager.dataflow.model.HIUReference;
```

Comment:
wildcar imports again.

Reply:
seems like some setting in intellij, will rectify that.